src/viam/ur/module/ur_arm.cpp

```diff
@@ -372,6 +375,18 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     current_state_->host = find_config_attribute<std::string>(cfg, "host");
     current_state_->speed.store(degrees_to_radians(find_config_attribute<double>(cfg, "speed_degs_per_sec")));
     current_state_->acceleration.store(degrees_to_radians(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2")));
+    try {
+        auto threshold = find_config_attribute<double>(cfg, "reject_move_request_threshold_deg");
+        if (threshold < 0 || threshold > 360) {
+            VIAM_SDK_LOG(error) << "reject_move_request_threshold_deg should be between 0 and 360, it is : " << threshold
+                                << " -  safety disabled";
+            reject_move_request_threshold_deg.reset();
+        } else {
+            reject_move_request_threshold_deg.emplace(threshold);
+        }
+    } catch (...) {
+        reject_move_request_threshold_deg.reset();
```

Comment:
I don't think you need the `reset`s if you move this to `struct state_`, because the `struct state_` lives or dies based on what happens in `configure_`